src/lib/make-toolbox-xml.js

```diff
@@ -4,7 +4,7 @@ const categorySeparator = '<sep gap="36"/>';
 
 const blockSeparator = '<sep gap="36"/>'; // At default scale, about 28px
 
-const motion = function (isStage, targetId) {
+const motion = function (_, isStage, targetId) {
```

Comment:
I know using `_` is going to suppress eslint warnings, but I think people won't remember what the variable is for if a category ever ends up needing initial setup. I think it would be better to use `isInitialSetup` everywhere and use `// eslint-disable-line no-unused-vars` if the linter complains.